flytekit/bin/entrypoint.py

```diff
@@ -262,6 +264,28 @@ def _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data
                 _handle_annotated_task(task_def, inputs, output_prefix, raw_output_data_prefix)
 
 
+@_scopes.system_entry_point
+def _execute_map_task(task_module, task_name, inputs, output_prefix, raw_output_data_prefix, max_concurrency, test):
+    task_module = _importlib.import_module(task_module)
+    task_def = getattr(task_module, task_name)
+
+    if not test and isinstance(task_def, PythonFunctionTask):
```

Comment:
does this also mean, we can now use Map task for everything? I dont think that will work. So for example PythonSparkFunctionTask us also derived from this. In the case we should just ensure this in the map-task compilation point

Reply:
sure, that's already done. this check is defensive in case someone mucks with the container args